privacy/activity.go

```diff
@@ -10,8 +11,8 @@ const (
 	ActivityReportAnalytics
 	ActivityTransmitUserFPD
 	ActivityTransmitPreciseGeo
-	ActivityTransmitUniqueRequestIds
-	ActivityTransmitTids
+	ActivityTransmitUniqueRequestIDs
+	ActivityTransmitTIDs
```

Comment:
Oh sorry about this...

Reply:
Not a problem. Just being picky about honoring Go acronym conventions (my IDE "yelled" at me here too).